docs/api/dt/fillna.rst

```diff
@@ -0,0 +1,157 @@
+
+.. xfunction:: datatable.fillna
+    :src: src/core/expr/fexpr_fillna.cc pyfn_fillna
+    :tests: tests/dt/test-fillna.py
+    :cvar: doc_dt_fillna
+    :signature: fillna(cols, direction)
+
+    .. x-version-added:: 1.1.0
+
+    For each column from `cols` fill the null values down or up with the previous/next non-null value. 
+
+    In the presence of :func:`by()`, the last non-null value is propagated down or up within each group.
+
+    Parameters
+    ----------
+    cols: FExpr
+        Input data for filling the nulls values down or up.
+
+    direction: string
```

Comment:
May be we call it `reverse` and make it `bool`? To be consistent with all the other places where we have the `reverse` parameter.

Reply:
Ahhhh nice pick up. Will do